core/claimmanager.go

```diff
@@ -220,7 +221,7 @@ func (c *BasicClaimManager) Claim() (claimCount int, rc chan types.Receipt, ec c
 		}
 
 		//Do the claim
-		go func(segRange [2]int64, rc chan types.Receipt, ec chan error) {
+		go func(rangeIdx int, segRange [2]int64, rc chan types.Receipt, ec chan error) {
```

Comment:
Making sure that the goroutine closes over the value at each iteration of the loop

Reply:
Is this used anywhere?